src/test/java/io/swagger/oas/test/BodyParamExtractionTest.java

```diff
@@ -80,7 +80,7 @@ public void testUUIDBodyParam() throws Exception {
                         .addMediaType("application/json",new MediaType()
                             .schema(new Schema().type("string").format("uuid"))));
 
-        JavaType jt = utils.getTypeFromRequestBody(body, definitions);
+        JavaType jt = utils.getTypeFromRequestBody(body, definitions)[0];
```

Comment:
same here

Reply:
I'm sure it will not be null on the test